### PR TITLE
Dev french

### PR DESCRIPTION
**Pull request by bertrandlalo**
This is a WIP PR for the french version of the study 🇫🇷✌️ 
--> Remaining todo : deployment. _But I'll need one of you to guide me with it, if possible_ 🙏

### CODE REVIEW

**Comment by fossecode** [comment]
Great work! We'll have it merged to master after merge conflicts are fixed, and then we will assist you with the deployment :)

File: app/views/privacy-statements/fr-lang-privacy-statement.ejs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    Nous utilisons des analytiques pour avoir un aperçu de l'utilisation du site Web. Nous utilisons
+  <a class="text-cvs-red hover:underline" href="https://simpleanalytics.com" target="_blank">Simple Analytics</a>
+  pour ça - un outil qui ne stocke pas de données personnelles et ne place pas de
+  cookies.
+</p>
+
+<h3>D'autres questions?</h3>
+<p>
+  Contactez nous sur Twitter:
+  <a class="text-cvs-red hover:underline" href="https://twitter.com/coronastatusNL" target="_blank"
```
Comment (fossecode) [inline app/views/privacy-statements/fr-lang-privacy-statement.ejs]: Should probably create a separate twitter account for France (or just change the line to an email address or just delete it) :)

File: app/domain/urls.ts [hunk 2]
```diff
@@ -9,7 +9,8 @@ export type Locale =
   | 'it'
   | 'tr'
   | 'dk'
-  | 'es-CO';
+  | 'es-CO'
+  | 'fr-FR'  ;
```
Comment (fossecode) [inline app/domain/urls.ts]: Please run `yarn lint` and fix any errors you see

Comment (bertrandlalo) [reply app/domain/urls.ts]: It should be ok now!

File: app/views/privacy-statements/fr-lang-privacy-statement.ejs (hunk 1 above)
Comment (bertrandlalo) [reply app/views/privacy-statements/fr-lang-privacy-statement.ejs]: Just created an account : https://twitter.com/CoronastatusFR

**Comment by fossecode** [comment]
Perfect, great work!

**Review by fossecode — APPROVED** [review]

**Pull request merged**